source/faq.rst

```diff
@@ -0,0 +1,243 @@
+and/or shared with Percona Support to solve an issue faster. You can get
+collected logs from PMM Client using the ``pmm-admin summary`` command. 
+Obtaining logs from PMM Server can be done `by specifying the
+``https://<address-of-your-pmm-server>/managed/logs.zip`` URL, or by clicking
+the ``server logs`` link on the `Prometheus dashboard <https://www.percona.com/doc/percona-monitoring-and-management/2.x/dashboards/dashboard-prometheus.html>`_:
+
+.. image:: .res/graphics/png/get-logs-from-prometheus-dashboard.png
+
+.. only:: showhidden
+
+	.. _metrics-resolution:
+
+	What resolution is used for metrics?
+	================================================================================
+
+	The |opt.mysql-metrics| service collects metrics with different resolutions (1
+	second, 5 seconds, and 60 seconds)
+
+	The |opt.linux-metrics| and |opt.mongodb-metrics| services are set up to collect
+	metrics with 1 second resolution.
+
+	In case of bad network connectivity between |pmm-server| and |pmm-client| or
+	between |pmm-client| and the database server it is monitoring, scraping every
+	second may not be possible when latency is higher than 1 second.  You can change
+	the minimum resolution for metrics by passing the ``METRICS_RESOLUTION``
+	environment variable when :ref:`creating and running the PMM Server container
+	<server-container>`.  To set this environment variable, use the ``-e`` option.
+	The values can be between *1s* (default) and *5s*.  If you set a higher value,
+	|prometheus| will not start.
+
+	For example, to set the minimum resolution to 3 seconds:
+
+	:command:`-e METRICS_RESOLUTION=3s`
+
+	.. note:: Consider increasing minimum resolution
+	   when |pmm-server| and |pmm-client| are on different networks,
+	   or when :ref:`pmm.amazon-rds`.
+
+.. only:: showhidden
+
+	.. _pmm.deploying.server.virtual-appliance.root-password.setting:
+
+	How to set the root password when |pmm-server| is installed as a virtual appliance
+	====================================================================================================
+
+	With your virtual appliance set up, you need to set the root password for your
+	|pmm-server|. By default, the virtual machine is configured to enforce changing
+	the default password upon the first login.
+
+	.. figure:: .res/graphics/png/command-line.login.1.png
+
+	   Set the root password when logging in.
+
+	Run your virtual machine and when requested to log in, use the following
+	credentials:
+
+	:User: root
+	:Password: percona
+
+	The system immediately requests that you change your password. Note that, for
+	the sake of security, your password must not be trivial and pass at least the
+	dictionary check. If you do not provide your password within sixty seconds you
+	are automatically logged out. In this case use the default credentials to log in
+	again.
+
+	.. figure:: .res/graphics/png/command-line.login.3.png
+
+	   Set a new password and have full access to your system
+
```

Comment:
But OVF is not availible for pmm2 for now

Reply:
ovf is also a hidden section which waits for its time